lib/actions/ui.js

```diff
@@ -389,3 +392,15 @@ export function setRouteViewerFilter(filter) {
     }
   }
 }
+
+/**
+ * Start over from the initial URL
+ */
+export function startOverFromInitialUrl() {
+  return function (dispatch, getState) {
+    const { initialUrl } = getState().otp
+    window.location.replace(initialUrl)
+    // FIXME: Trigger a reload to reset page, otherwise the UI keeps some previous states.
```

Comment:
I think you can delete this comment line - page reload seems to be working well

Reply:
I combined with the JsDoc of this method in https://github.com/opentripplanner/otp-react-redux/pull/574/commits/090c0aa72dd1b5c23858c77fa101497e47b509c9.